### PR TITLE
Setup GroupDeleteCommand class

### LINKED ISSUE #239: Set up GroupDeletecommand class
Include necessary constructors, attributes, methods and Javadocs

### PR DESCRIPTION
**Pull request by lelerer**
Fixes #239

### CODE REVIEW

**Review by froststein — APPROVED** [review]
Looks good to merge

File: src/main/java/seedu/splitlah/command/GroupDeleteCommand.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    public GroupDeleteCommand(int groupId) {
+        // TODO: Add assert to check whether groupId is valid.
+        this.groupId = groupId;
+    }
+
+    /**
+     * Prepares user arguments for the creation of an GroupDeleteCommand object.
+     *
+     * @param commandArgs The user's arguments.
+     * @return An GroupDeleteCommand object if necessary parameters were found in user arguments,
```
Comment (warrencxw) [inline src/main/java/seedu/splitlah/command/GroupDeleteCommand.java:41]: Consider correcting `An` to `A` to rectify minor grammar mistake.

**Review by warrencxw — APPROVED** [review]
Everything looks good, but consider making this minor change to fix a grammar error.

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Pull request merged**